R/use_extendr.R

```diff
@@ -5,9 +5,10 @@
 #' `"Hello world!"`. It also adds wrapper code so this Rust function can be called from
 #' R with `hello_world()`.
 #'
-#' To avoid possibly messing up your R package, `use_extendr()` will not do anything if
-#' either a directory `src` or a file `R/extendr-wrappers.R` is already present in your
-#' package source.
+#' This function can be called on an existing project;
+#' you will be asked if either the `src` directory or the `R/extendr-wrappers.R` file
+#' are existing. And, even if you select to overwrite them, `Cargo.toml`, `lib.rs`
+#' and `extendr-wrappers.R` will not be overwritten.
```

Comment:
It is a little bit unclear to me what are we achieving with this? Like, if nothing exists, it will be the same initialization procedure, but if there are rust files, we do not touch them? So if someone modified Makevars, it will be lost? Maybe it is worth mentioning what **will be** modified instead of what will remain untouched?

Reply:
If using a version control system such as Git, I wonder if it is worth listing them all here, since changes can be detected by the version control system.

Reply:
`use_template` will ask the user individually whether or not to overwrite the file, and will not do so without the user's permission.